### PR TITLE
Bump cloud version to v13.1.0

### PR DESCRIPTION
**Pull request by sclevine**
Update docs to Cloud version v13.1.0.

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]
Aren't we about to go to 13.1 though?

**Comment by sclevine** [comment]
Not for any existing customers CC: @jimbishopp

**Comment by sclevine** [comment]
Changed course to v13.1.0, even though existing enterprise users are not being upgraded to v13 at this time.

File: docs/config.json [hunk 1]
```diff
@@ -1697,7 +1697,7 @@
       "aws_secret_access_key": "zyxw9876-this-is-an-example"
     },
     "cloud": {
-      "version": "12.4.5",
+      "version": "13.1.0",
       "major_version": "12",
```
Comment (zmb3) [inline docs/config.json:1701]: Change `major_version` too then?

Comment (sclevine) [reply docs/config.json]: Ah, thanks!

**Review by ptgott — APPROVED** [review]

**Pull request merged**